### DISCUSSION TITLE
Unable to use the react graphic-walker component in Angular

### DISCUSSION THREAD

**Discussion by sriramgc** [Kanaries/graphic-walker#299] [Q&A]
Hi Team,

I am unable to embed the react component inside angular.I have used the following article for ref:
https://medium.com/@zacky_14189/embedding-react-components-in-angular-the-easy-way-60f796b68aef
but getting exceptions with the following error:

Error: node_modules/@kanaries/graphic-walker/dist/components/toolbar/components.d.ts:7:68 - error TS2694: Namespace '"C:/2023/ss/ss/node_modules/styled-components/dist/index"' has no exported member 'StyledComponent'.

7 export declare const ToolbarContainer: import("styled-components").StyledComponent<"div", any, {

Could you please help or any advise to resolve this .

**Comment by jojocys** [comment]
For current version (before <= 0.4.55), `styled-components` are used as peer-deps, you need to install styled-components by yourself.
